test/presubmit-tests.sh

```diff
@@ -54,8 +55,12 @@ install-node() {
 
 ui-unittest(){
     install-node
+    npm install
```

Comment:
Not sure if we need `npm install` before unit test

Reply:
yes it is needed otherwise npm test fails